micro-rdk-nmea-macros/src/attributes.rs

```diff
@@ -46,6 +46,7 @@ pub(crate) struct MacroAttributes {
     pub(crate) is_lookup: bool,
     pub(crate) is_fieldset: bool,
     pub(crate) length_field: Option<Ident>,
+    pub(crate) pgn: Option<u32>,
```

Comment:
It isn't really optional though, is it? Is there a check somewhere that ensures that this gets populated into `MacroAttributes`? I mean, I see why you did it this way, so you can make `MacroAttributes` and then populate it step-by-step, so I don't mind that much. I guess you could work around it with a builder that had an `Option` and then a built type that didn't. But that feels like overkill to just remove on `Option`

Reply:
An instance of `MacroAttributes` pertains to a particular field, so it should, in fact, be optional